src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,14 +189,23 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string errorText = ProcessResources.NoProcessFoundForGivenName;
+                    string errorName = "NoProcessFoundForGivenName";
+
+                    if (int.TryParse(pattern, out int x) && x > 0)
```

Comment:
Maybe it should be `x >= 0`?

Reply:
(I thought we want hide system process)

Reply:
It seems like if `Get-Process -Id 0` would succeed, then we want to have the new error message here